src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,8 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $serviceIdToMethodNameMap = array();
+    private $usedMethodNames = array();
```

Comment:
this one should be initialized with all method names of the base container class (and any parent), to avoid issues